src/components/checkout/shipping-methods.js

```diff
@@ -16,13 +16,28 @@ class ShippingMethods extends Component {
     this.ShippingMethodsHeader = componentMapping('ShippingMethodsHeader')
   }
 
+  /**
+   * Renders the continue button
+   */
+  renderContinueButton () {
+    const { handleFormSubmit, isThirdPartyPayment} = this.props
+    const label = (isThirdPartyPayment ? 'Review Your Order' : 'Continue To Payment')
```

Comment:
you shouldnt need the brackets
```suggestion
    const label = isThirdPartyPayment ? 'Review Your Order' : 'Continue To Payment'
```